roles/spark/tasks/start-spark.yml

```diff
@@ -29,4 +28,4 @@
 
 - name: Spark-shuffle started
   systemd: name=spark-shuffle state=started daemon_reload=yes
-  when: inventory_hostname in groups['spark-worker'] and inventory_hostname not in groups['spark-master']
+  #when: inventory_hostname in groups['spark-worker'] and inventory_hostname not in groups['spark-master']
```

Comment:
Should shuffle now run on all nodes where start-spark task is run on?
If so please remove comment.